quokka/core/content/models.py

```diff
@@ -230,13 +230,13 @@ def metadata(self):
         # TODO: get metadata from database
         # TODO: implement libratar/gravatar
         # return {
-            # 'cover': 'foo',
```

Comment:
Please, remove the comet lines

Reply:
removed and created a new issue to dont lost these itens:
https://github.com/rochacbruno/quokka/issues/681

Reply:
commit => https://github.com/rochacbruno/quokka/pull/680/commits/d0a448bf234361980be34bc2ed345c94b266705b